src/dialects/abstract/query-generator.js

```diff
@@ -1216,11 +1216,11 @@ export class AbstractQueryGenerator {
    * @param   {string}               _column   The JSON column
    * @param   {string|Array<string>} [_path]   The path to extract (optional)
    * @param   {boolean}              [_isJson] The value is JSON use alt symbols (optional)
-   * @returns {string}                        The generated sql query
+   * @returns {Error|string}                   The generated sql query
```

Comment:
It never returns an error

```suggestion
   * @returns {string}                   The generated sql query
```

Reply:
Yeah, I wasn't too sure about this change when I looked at it again. I think my original thought was that if it's not implemented it throws an Error, but that's not what it returns